# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/shared-mime-info/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

../src/update-mime-database.cpp
../src/update-mime-database.cpp:2161:28: error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
 2161 |                 env = "/usr/local/share/"PATH_SEPARATOR"/usr/share/";
      |                                          ^

--- shared-mime-info-2.3/src/update-mime-database.cpp.vanilla	2023-10-10 09:57:23.219242710 +0200
+++ shared-mime-info-2.3/src/update-mime-database.cpp	2023-10-10 09:57:55.558242816 +0200
@@ -2158,7 +2158,7 @@
 
 	env = getenv("XDG_DATA_DIRS");
 	if (!env)
-		env = "/usr/local/share/"PATH_SEPARATOR"/usr/share/";
+		env = "/usr/local/share/" PATH_SEPARATOR "/usr/share/";
 	dirs = g_strsplit(env, PATH_SEPARATOR, 0);
 	g_return_if_fail(dirs != NULL);
 	for (n = 0; dirs[n]; n++)
